05_ANALYSIS_TOOLS/etl_pipeline/config/data_sources.yaml

```
@@ -0,0 +1,102 @@
+# Data Sources Configuration
+# Defines all data sources for the ETL pipeline
+
+data_sources:
+  sales:
+    type: csv
+    path: "../02_OPERATIONAL_DATA/Sales_Data/report-371-1755757545-sales.csv"
+    encoding: utf-8
+    delimiter: ","
+    has_header: true
+    description: "Main sales transaction data"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)